test/source/mock/strategies/send-message-strategy.ts

```diff
@@ -4,6 +4,26 @@ import { HttpClientErr } from '../api.js';
 import { PgpMsg, Pgp } from "../../core/pgp.js";
 import { Buf } from '../../core/buf.js';
 import { Config } from '../../util/index.js';
+import { GmailMsg, Data } from '../data.js';
+
+class AddMessageToStorageTestStrategy implements ITestMsgStrategy {
```

Comment:
It's not so clear that this should be a test strategy, it should rather just be a method on `Data`? What is it testing?

Reply:
Now I see how it's used, this should be renamed to `PwdEncryptedMessageStrategy` then.